fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/JobsServletReactiveChainforResultsEndpointSpec.groovy

```diff
@@ -46,13 +47,15 @@ class JobsServletReactiveChainforResultsEndpointSpec extends Specification {
     PreResponseStore mockPreResponseStore
     JobsServlet mockJobServlet
     HttpResponseMaker httpResponseMaker
+    ResponseWriter responseWriter
 
     def setup() {
         objectMappersSuite = Mock(ObjectMappersSuite)
         ObjectMapper objectMapper = Mock(ObjectMapper)
         ObjectWriter objectWriter = Mock(ObjectWriter)
         objectMappersSuite.getMapper() >> objectMapper
         objectMapper.writer() >> objectWriter
+        responseWriter = Mock(ResponseWriter)
```

Comment:
I would probably just build a simple ResponseWriter here (maybe the `JsonResponseWriter`). I generally prefer to use mocks only when it's either impossible, or a truly MASSIVE pain to build an actual object, and that object isn't pertinent to the test. Since it's pretty simple to build a ResponseWriter, I don't see any reason not to use a real one.